client/src/images/star.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?>
```

Comment:
Hi @cynthiaeddy! Did you mean to add this file a second time (there is one already in the public directory) to the repo or is this a different star.svg?

Reply:
i moved the star svg to the images folder(which should be placed inside an assets folder)  bc I was having trouble directly accessing it, though I know it works in other components. Should I import it instead from the public folder?

Reply:
here's what it looked like before i moved the svg to the images folder.
<img width="778" alt="Screen Shot 2021-12-13 at 2 16 54 PM" src="https://user-images.githubusercontent.com/15850286/145874251-a8ec67d2-c6cc-4c6b-a2c0-4917861fb05c.png">
here's the code:
<img width="500" alt="Screen Shot 2021-12-13 at 2 19 02 PM" src="https://user-images.githubusercontent.com/15850286/145874439-4447d4f2-73b0-4977-b7ce-cd450d2887de.png">